core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceRepository.scala

```diff
@@ -16,9 +27,20 @@ import scala.concurrent.Future
   */
 class WorkspaceRepository(dataSource: SlickDataSource) {
 
-  def getWorkspace(workspaceId: UUID): Future[Option[Workspace]] =
+  def getWorkspace(workspaceId: UUID): Future[Option[Workspace]] = getWorkspace(workspaceId, None)
+
+  def getWorkspace(workspaceId: UUID, attributeSpecs: Option[WorkspaceAttributeSpecs]): Future[Option[Workspace]] =
     dataSource.inTransaction { access =>
-      access.workspaceQuery.findById(workspaceId.toString)
+      access.workspaceQuery.findV2WorkspaceById(workspaceId, attributeSpecs)
```

Comment:
I don't understand… the previous method did not say it was for v2 workspaces.

Reply:
There two parts to the refactor:
1. Eliminating the user-facing queries that return v2 workspaces. Included in this was removing the 'v2' component of the name, since it is now redundant.
2. The `getV2WorkspaceContext` (using the id) wasn't actually using the v2 specific query.

Reply:
OK, I'm still having trouble understanding the diff, but I agree that we only want to return v2 workspaces now.